tools/replay/parse_dump_resources_cli.cpp

```diff
@@ -159,6 +160,163 @@ static bool CheckIndicesForErrors(const gfxrecon::decode::VulkanReplayOptions& v
     return false;
 }
 
+static VkImageAspectFlags StrToImageAspectFlagBits(const std::string& str)
+{
+    VkImageAspectFlags flags = VK_IMAGE_ASPECT_NONE;
+
+    if (str.find("VK_IMAGE_ASPECT_COLOR_BIT") != std::string::npos)
```

Comment:
Please change this to a  static `std::map<std::string, VkImageAspectFlags>` and a loop.